gen/lib/gen/commands/help.rb

```diff
@@ -0,0 +1,21 @@
+require 'gen'
+
+module Gen
+  module Commands
+    class Help < Gen::Command
+      def call(_args, _name)
+        puts CLI::UI.fmt("{{bold:Available commands}}")
+        puts ""
+
+        Gen::Commands::Registry.resolved_commands.each do |name, klass|
+          next if name == 'help'
+          puts CLI::UI.fmt("{{command:#{Gen::TOOL_NAME} #{name}}}")
```

Comment:
Should this not be inside the `if`?

Reply:
I'm not sure what you mean. Maybe you misread line 11?

Reply:
I mean

```ruby
if help = klass.help
  puts CLI::UI.fmt("{{command:#{Gen::TOOL_NAME} #{name}}}")
  puts CLI::UI.fmt(help)
  puts ""
end
```

Why output the command unless there's help to be had for it?